service/role.go

```diff
@@ -253,15 +253,6 @@ func (r *roleService) RefreshRoleTokenCache(ctx context.Context) <-chan error {
 	return echan
 }
 
-// handleExpiredHook is a handler function for gache expired hook
-func (r *roleService) handleExpiredHook(ctx context.Context, key string) {
-	glg.Warnf("handleExpiredHook(), key: %s", key)
```

Comment:
better to keep the warning log?
I think we don't expect any tokens to be expired in cache.

Reply:
this function is removed so...

Reply:
https://github.com/yahoojapan/athenz-client-sidecar/pull/23#discussion_r338855030